### ISSUE TITLE
Re-implementation of Cycle?

### ISSUE THREAD

**Issue by philcockfield** [pH200/cycle-react#34]
Is this a re-implementation of cycle.js for React, or does it act as a Driver for the original Cycle.js?

It looks like it's the former.  Is this because a driver (like the [official react-native driver for Cycle](https://github.com/cyclejs/cycle-react-native)) doesn't make sense?

Sorry if this is a wrong-headed question, I'm just trying to get my head around what I might be missing if I'm going down a "cycle" type path, but not using the core cycle library.  Cycle looks great, but I really don't want to abandon the HUGE eco-system of React proper.

Thanks

**Comment by pH200** [comment]
This is the implementation of Cycle.js for React. However, it was before Cycle.js introduced the Driver API. Since then the API design of Cycle.js has changed. So, Cycle-React doesn't look like Cycle.js a lot right now.

By the way, Cycle-React could act [as a Driver](https://github.com/pH200/cycle-react/blob/v2.0.0/docs/cycle-js-driver.md) before. However, this feature was removed because:

1. React can and **must** update DOM by itself (ReactDOM.render), unlike [virtual-dom](https://github.com/Matt-Esch/virtual-dom), you diff and patch the DOM by yourself.
2. We decided to abandon the selector API from Cycle.js, because it doesn't act smoothly with React components.

So, if you don't want to abandon the eco-system of React. I would definitely suggest you check out Cycle-React. On the other hand, if you want to try something which is not React but still a awesome framework for front-end development, then Cycle.js is the answer.
